client/us_census_client.py

```diff
@@ -0,0 +1,27 @@
+import aiohttp
+import asyncio
+import async_timeout
+import os
+from typing import Dict, Union, List
+
+from client.us_census_variable_names import (
+    TOTAL_HOUSEHOLDS_RECEIVING_FOOD_STAMPS_IN_LAST_YEAR, 
+    ILLINOIS_STATE_CODE
+)
+
+async def get_request(session: aiohttp.ClientSession, url: str, params: Dict[str, str]) -> Union[List, Dict]:
```

Comment:
I'm not really familiar with using async requests here. Is this necessary to do?

Reply:
It's not necessary, we could use `requests`, which is a synchronous library. But this will be faster when we need to make more than one API request, since we can send them all at once instead of one after another. Thoughts?

Reply:
Gotcha. Is there anything like promises in javascript for python? I know that sometimes doing async stuff can get pretty hairy pretty quickly if we're (mostly I) am not familiar with it.

Reply:
Sounds like these "coroutines" in python are pretty similar to javascript promises: https://stackabuse.com/python-async-await-tutorial/

Reply:
I don't know javascript though